lms/templates/navigation.html

```diff
@@ -175,20 +176,26 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         </%block>
       </ol>
 
+      <%
+        query_params = urlencode(request.GET)
+        query_params = "?%s" % query_params
```

Comment:
I would write these two lines in one `query_params = "?%s" % urlencode(request.GET)`